docs/details/shared-folders.md

```diff
@@ -27,11 +23,4 @@ user must be able to `ssh` to `localhost`. That is `ssh localhost` should work.
 This can probably be achieved by simply adding the contents of `~/.ssh/id_rsa.pub`
 to `~/.ssh/authorized_keys`.
 
-`transient` also requires that the guest image have `sshfs` installed or be an
-operating system `transient` knows how to 'provision'. Currently the following
-operating systems can be provisioned by `transient`:
-
-- CentOS 7
-- Red Hat Enterprise Linux 7
-
-Other `yum` based distributions may also work.
+`transient` also requires that the guest image have `sshfs` installed.
```

Comment:
Maybe a slightly different wording:

     `transient` also requires that the guest image have `sshfs` installed to use this feature. To 
    build a guest image with `sshfs` pre-installed, see [Building Images](../images/building)

Reply:
Done.